README

```diff
@@ -55,6 +54,6 @@ Binaries for NOIR (musl) -> https://github.com/kiss-community/repo-bin
 
 * Modify `KISS_PATH` such that the `bin` repository takes priority over other repositories:
 
-$ export KISS_PATH=/path/to/main/bin:$KISS_PATH
```

Comment:
Hey @mwelwankuta , this line isn't supposed to be back ticked because it only serves as a frame of reference for setting the `KISS_PATH`.

Reply:
thank you for feedback, i love what you are working on, one day i might start something similar 😅.

Reply:
Thanks for your interest in NOIR, iI really appreciate it.
I look forward to seeing what you come up with.